code/__DEFINES/flags.dm

```diff
@@ -73,6 +73,8 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 #define BLOCK_SUICIDE				(1<<8)
 /// Can the Xenobio management console transverse this area by default?
 #define XENOBIOLOGY_COMPATIBLE		(1<<9)
+/// Are hidden stashes allowed to spawn here?
+#define HIDDEN_STASH_LOCATION		(1<<10)
```

Comment:
do areas have a relational flag define for area_flags (so they can be varedited?)

if so add it to that as well